sgpt/client.py

```diff
@@ -47,18 +48,29 @@ def _request(
             "model": model,
             "temperature": temperature,
             "top_p": top_probability,
+            'stream': True,
         }
         endpoint = f"{self.api_host}/v1/chat/completions"
         response = requests.post(
-            endpoint, headers=headers, json=data, timeout=REQUEST_TIMEOUT
+            endpoint, headers=headers, json=data, timeout=REQUEST_TIMEOUT, stream=True
         )
         response.raise_for_status()
-        return response.json()
+        for line in response.iter_lines():
+            data = line.lstrip(b'data: ').decode('utf-8')
+            if data == '[DONE]':
+                break
+            if not data:
+                continue
+            data = json.loads(data)
+            delta = data['choices'][0]['delta']
+            if 'content' not in delta:
+                continue
+            yield delta['content']
```

Comment:
Please use `"` instead of `'` since CI pipeline will check it with black and fail.